src/main/java/com/gluonhq/richtext/RichTextAreaSkin.java

```diff
@@ -583,6 +589,14 @@ private MenuItem createMenuItem(String text, ActionCmd actionCmd) {
         return menuItem;
     }
 
+    private double getMaxWidthAvailable() {
+        final double viewportWidth = scrollPane.getViewportBounds().getWidth();
+        final double contentAreaWidth = getSkinnable().getContentAreaWidth();
+        final double contentAreaLimit = contentAreaWidth == 0 ? viewportWidth : contentAreaWidth;
+        final double padding = textFlow.getPadding().getLeft() + textFlow.getPadding().getRight() + root.getPadding().getLeft() + root.getPadding().getRight();
+        return Math.min(contentAreaLimit, viewportWidth) - padding - 1;
```

Comment:
Does `-1` has a role to play here?

Reply:
Helps having a smaller imageView that doesn't push the textFlow, avoiding an infinite relayout as scrollbars are needed or not.